src/store/reducers/appReducer.js

```diff
@@ -108,6 +109,7 @@ export const getVersion = state => getAppReducer(state).version;
 export const getTheme = state => getAppReducer(state).theme;
 export const getDeviceGroupEntities = state => getAppReducer(state).deviceGroups;
 export const getActiveDeviceGroupId = state => getAppReducer(state).activeDeviceGroupId;
+export const getAzureMapsKey = state => getAppReducer(state).azureMapsKey;
```

Comment:
I'm assuming that you still need to write the code which fetches the map key from backend, correct?

Reply:
Correct.